docs/CONFIG-VARS.md

```diff
@@ -191,22 +190,49 @@ When setting V4_CFG_TLS_MODE to a value other than "disabled" and no V4_CFG_TLS_
 
 ## Postgres
 
-| Name | Description | Type | Default | Required | Notes | Tasks |
-| :--- | ---: | ---: | ---: | ---: | ---: | ---: |
-| V4_CFG_POSTGRES_TYPE | Postgres installation type | string | | true | [internal,external] | viya |
+Postgres servers can be defined with the postgres_servers variable which is a map of objects. The variable has the following format:
+
+```bash
+V4_CFG_POSTGRES_SERVERS:
+  default: {}
+  ...
+```
 
-### External Postgres
+**NOTE**: the `default` elements is always required . This will be the default server. Below is the list of parameters each element can contain.
```

Comment:
I like a comment thomas used in the viya4-iac-* text. Since you added this capability this morning can we add this statement or something similar

Each server element, like foo = {}, can contain none, some, or all of the parameters listed below.